actions_test.go

```diff
@@ -486,7 +486,7 @@ snapshot:
 					"properties":  map[string]interface{}{},
 				}}}},
 	}, {
-		description: "A simple snapshot actions YAML with numeric characters.",
+		description: "A simple snapshot actions YAML ending characters.",
```

Comment:
I'm not sure about this edit? The description no longer makes sense....it is maybe 'with ending' ?

Reply:
Added `with names` so it's more clear.